datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -4,10 +4,13 @@ keywords: docker, registry, high-availability, backup, recovery
 title: DTR backups and recovery
 ---
 
-When you use Docker Trusted Registry on a production setting, you should first
-configure it for high availability.
-
-The next step is creating a backup policy and disaster recovery plan.
+DTR relies on consensus to operate and if it loses consensus permanently
```

Comment:
When you say "DTR relies on consensus to operate" it's very clear in your head what you mean, but people with less experience in distributed systems might not really understand what you mean. So you need to provide a bit more context. What about

The way multiple DTR instances replicate data is by using log consensus. When you data is changed in one of the instances, it its replicated into the other instances, but is only persisted if a majority or replicas acknowledges the change.
This means that if the majority of the replicas are unhealthy, DTR data cannot be replicated, and you'll have to restore the DTR data from an existing backup.

The rest of the sentence is not really important for the first part of the explanation, we can move it down below.